spec/rails_helper.rb

```diff
@@ -46,10 +46,12 @@
   RedisLockDb.redis = redis
 
   config.before(:suite) do
+    TradeTariffBackend.redis.flushdb
     redis.flushdb
```

Comment:
Hmm, so whose instance is this?

Reply:
A separate instance for creating a mutex I believe

Reply:
Have you tried removing it does it break the tests?

Reply:
It's using a different redis database from the default (0) one

Reply:
Quite a few tests fail if we remove clearing the lock db